.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: minimal
+
+git:
+  depth: 1
```

Comment:
Did we say we might up this to a greater depth? I can't remember

Reply:
I think it's fine as 1, it's only caused occasional issues when we commit in short succession on the same branch